src/nix-build/nix-build.cc

```diff
@@ -67,6 +67,19 @@ std::vector<string> shellwords(const string & s)
     return res;
 }
 
+static string shellquote(string s) {
```

Comment:
There is a function named `shellEscape` in `util.cc` that appears to do the same thing.

Reply:
Resolved.